interface/forms/newGroupEncounter/common.php

```diff
@@ -279,7 +279,7 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
  if ($sensitivities && count($sensitivities)) {
   usort($sensitivities, "sensitivity_compare");
 ?>
-     <td class='bold' nowrap><?php echo xlt('Sensitivity'); ?>:</td>
+     <td class='bold' nowrap><?php echo xlt('Sensitivity:'); ?>:</td>
```

Comment:
Right I'm agree but in the same form for patient it exists with colons ([https://github.com/openemr/openemr/blob/master/interface/forms/newpatient/common.php#L285](https://github.com/openemr/openemr/blob/master/interface/forms/newpatient/common.php#L285))
and I didn't want to fix in the patient's form because there isn't constant without colons..
Now I only removed the outside colons.